scipy/stats/_covariance.py

```diff
@@ -358,6 +358,54 @@ def whiten(self, x):
         """
         return self._whiten(np.asarray(x))
 
+    def colorize(self, x):
+        """
+        Perform a colorizing transformation on data.
+
+        "Colorizing" ("color" as in "colored noise", in which differenct
+        frequencies may have different magnitudes) transforms a set of
+        uncorrelated random variables into a new set of random variables with
+        the desired covariance. When a coloring transform is applied to a
+        sample of points distributed according to a multivariate normal
+        distribution with identity covariance and zero mean, the covariance of
+        the transformed sample is approximately the covariance matrix used
+        in the coloring transform.
+
+        Parameters
+        ----------
+        x : array_like
+            An array of points. The last dimension must correspond with the
+            dimensionality of the space, i.e., the number of columns in the
+            covariance matrix.
+
+        Returns
+        -------
+        x_ : array_like
+            The transformed array of points.
+
+        References
```

Comment:
Can you please also add https://www.projectrhea.org/rhea/index.php/ECE662_Whitening_and_Coloring_Transforms_S14_MH? It is cited in Wikipedia and is much more helpful.

Reply:
I'm a little hesitant because *looks* a little less reputable than Wikipedia (which is a low bar.. even though Wikipedia cites it), and I didn't use it. "Coloring transform" is defined in the Wikipedia article, and based on that it must be the inverse of the whitening transform, so I thought that was enough. But if you ask again I'll add it : )

Reply:
It might be useful for someone unaware of the concept since Wikipedia just writes one line about it and links to the article anyways. Also, I wasn't able to find anything else that talks about it in detail. If you can, feel free to add it instead!

Reply:
Added in next commit: https://www.researchgate.net/publication/331063251_Generalization_of_Coloring_Linear_Transformation/link/5c63d0f345851582c3e42610/download

Reply:
Thanks for lookin into this. The reference looks good!